tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -534,23 +571,36 @@ class LineChart {
   }
 
   private resetXDomain() {
-    // (Copied from DragZoomLayer.unzoom.)
-    const xScale = this.xScale as any;
-    xScale._domainMin = null;
-    xScale._domainMax = null;
-    const xDomain = xScale._getExtent();
+    let xDomain;
+    if (this._defaultXRange) {
+      // Use the range specified by the caller.
+      xDomain = d3.scaleLinear().domain(this._defaultXRange).domain();
```

Comment:
Hmm, how is `d3.scaleLinear().domain(x).domain()` different from `x`?

Reply:
Ah, indeed, just using the list works. Done.